nikola/nikola.py

```diff
@@ -1031,7 +1031,7 @@ def init_plugins(self, commands_only=False, load_all=False):
         extra_plugins_dirs = self.config['EXTRA_PLUGINS_DIRS']
         self._loading_commands_only = commands_only
         self._plugin_places = [
-            resource_filename('nikola', 'plugins'),
+            resources.files('nikola').joinpath('plugins'),
```

Comment:
`resources.files()` has been added in Python 3.9 (https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files), while this repo still supports Python 3.8 (https://github.com/getnikola/nikola/blob/master/setup.py#L142).

Reply:
> `resources.files()` has been added in Python 3.9 (https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files), while this repo still supports Python 3.8 (https://github.com/getnikola/nikola/blob/master/setup.py#L142).

should we use importlib_resources instead of importlib.resources ?

Reply:
No. Please use the built-in `importlib.resources`, but only the functions available in Python 3.7: https://docs.python.org/3.7/library/importlib.html#module-importlib.resources

Reply:
You can always make code depend on `sys.version`.